example/server.js

```diff
@@ -110,6 +110,19 @@ function init (config, callback) {
           },
           {
             method: 'GET',
```

Comment:
It's just the example, but it might be nicer to use a post request with the password pass in the payload of the request instead as a query parameter